src/bokeh/models/tickers.py

```diff
@@ -78,6 +82,78 @@ class Ticker(Model):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
+class CustomJSTicker(Ticker):
+    ''' Generate tick locations that are computed by a user-defined function.
+
+    A ``CustomJSTicker`` may be used with either a continuous (numeric) axis,
+    or a categorical axis. However, only basic, non-hierarchical categorical
+    axes (i.e. with a single level of factors) are supported.
+
+    .. warning::
+        The explicit purpose of this Bokeh Model is to embed *raw JavaScript
+        code* for a browser to execute. If any part of the code is derived
+        from untrusted user inputs, then you must take appropriate care to
+        sanitize the user input prior to passing to Bokeh.
+
+    '''
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    args = Dict(String, AnyRef, help="""
+    A mapping of names to Python objects. In particular those can be bokeh's models.
+    These objects are made available to the formatter's code snippet as the values of
```

Comment:
```suggestion
    These objects are made available to the formatters code snippet as the values of
```

Reply:
The possessive seems correct to me, the core snippet belongs to ("is a property of") the object. Except the object is a ticker, not a formatter
```suggestion
    These objects are made available to the ticker's code snippet as the values of
```